trial-config-schema/trial-json-schema.json

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
In this repo folders have special meaning, so creating a folder for this file isn't consistent with that...
What about putting it on a ".ci" folder or in the root?

Reply:
Agree, moved the schema file to .ci folder